### ISSUE TITLE
P3D Rendering Blank

### ISSUE THREAD

**Issue by sampottinger** [benfry/processing4#121]
Looking into this now... weirdly I don't see it happening on the integration branch over at https://github.com/sampottinger/processing4/. 

*Edit:* It turns out that it fails when bumping past JDK 11.0.6. Reverting to 11.0.6 restores expected behavior.

<img width="1017" alt="Screen Shot 2020-08-16 at 8 13 42 PM" src="https://user-images.githubusercontent.com/110391/90354078-001f6980-dffd-11ea-900c-936c3a70dab6.png">

**Comment by sampottinger** [comment]
Found on Mac 10.14.6

**Comment by sampottinger** [comment]
Could not reproduce on Windows

**Comment by sampottinger** [comment]
I have a hunch it's vaqua related but `editor.allow_vaqua = false` on Mac doesn't change the behavior... Looking into it.

**Comment by sampottinger** [comment]
Disabling the vaqua code doesn't seem to do it... looking into other possible culprits.

**Comment by sampottinger** [comment]
Hey @benfry, still confirming this but it looks this may be an issue with https://github.com/processing/processing4/commit/b8de4b85a821f2279383d96d7a80f09408651637 or somewhere near there...

**Comment by sampottinger** [comment]
Confirmed that the issue is _not_ reproducible on mac with `fix_mac_os_save` which introduces vaqua.

**Comment by sampottinger** [comment]
Humm... actually

```
$ git checkout 12205d473691460caa25f771016125088430360a
$ ant clean; ant run
```

seems to reproduce (12205d473691460caa25f771016125088430360a). However, 25f67253d21e93a8a95b4fb27e744f6f0ce6f315 works fine. Digging in further...

**Comment by sampottinger** [comment]
Oh golly gee whiz 😑. Ummm it looks like it happens when bumping the JDK above 11.0.6. Reverting fixes see #123. @benfry

**Comment by benfry** [comment]
Hooray for progress
